todo_app/templates/index.html

```diff
@@ -11,8 +11,18 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
+
+        {% for item in items %}
+        <li>{{ item.title }}</li>
+        {% endfor %}
+
         <!-- List the todo items here -->
       </ul>
     </div>
   </div>
+
+  <form method="post">
```

Comment:
You can make the purpose of the form clearer by setting e.g. `action=/add_item` and then change the route to `'/add_item'` to match.

Makes the HTTP request's intention clearer and also means you can have multiple forms on one page.